TeachingRecordSystem/src/TeachingRecordSystem.Core/Services/TrsDataSync/TrsDataSyncService.cs

```diff
@@ -61,6 +61,7 @@ internal async Task ProcessChangesAsync(CancellationToken cancellationToken)
         var modelTypesToSync = optionsAccessor.Value.ModelTypes;
 
         // Order is important here; the dependees should come before dependents
+        await SyncIfEnabledAsync(TrsDataSyncHelper.ModelTypes.Induction);
```

Comment:
Shouldn't this come after `Person`?

Reply:
It's before to avoid the induction status changing by itself in the contact record if there are also changes in the dfeta_induction table i.e. where maybe start date / end date should change too